[invalid_tentative.py]
  [test_params_function_declaration_invalid_type[None\]]
    expected: FAIL

  [test_params_function_declaration_invalid_type[False\]]
    expected: FAIL

  [test_params_function_declaration_invalid_type[42\]]
    expected: FAIL

  [test_params_function_declaration_invalid_type[function_declaration3\]]
    expected: FAIL

  [test_params_function_declaration_invalid_type[function_declaration4\]]
    expected: FAIL

  [test_params_arguments_invalid_type[False\]]
    expected: FAIL

  [test_params_arguments_invalid_type[SOME_STRING\]]
    expected: FAIL

  [test_params_arguments_invalid_type[42\]]
    expected: FAIL

  [test_params_arguments_invalid_type[arguments3\]]
    expected: FAIL

  [test_params_sandbox_invalid_type[False\]]
    expected: FAIL

  [test_params_sandbox_invalid_type[42\]]
    expected: FAIL

  [test_params_sandbox_invalid_type[sandbox2\]]
    expected: FAIL

  [test_params_sandbox_invalid_type[sandbox3\]]
    expected: FAIL
